File: Desafio/src/br/com/ibm/assassino/Crime.java
package br.com.ibm.assassino;
public class Crime {
	
	static String[] suspeitos = {"Charles B. Abbage", "Donald Duck Knuth", "Ada L. Ovelace", "Alan T. Uring", "Ivar J. Acobson", "Ras Mus Ler Dorf"};
	
	static String[] locais = {"Redmond", "Palo Alto", "San Francisco", "Tokio", "Restaurante no Fim do Universo", "São Paulo", "Cupertino", "Helsinki", "Maida Vale", "Toronto"};
	
	static String[] armas = {"Peixeira", "DynaTAC 8000X", "Trezoitão", "Trebuchet", "Maça", "Gládio"};	
	
	
	public static String[] getSuspeitos() {
		return suspeitos;
	}

	public static void setSuspeitos(String[] suspeitos) {
		Crime.suspeitos = suspeitos;
	}

	public static String[] getLocais() {
		return locais;
	}

	public static void setLocais(String[] locais) {
		Crime.locais = locais;
	}

	public static String[] getArmas() {
		return armas;
	}

	public static void setArmas(String[] armas) {
		Crime.armas = armas;
	}
	
	

}

File: assassino/bin/crime.js

function getSuspeitos(){
    const suspeitos = []

    suspeitos.push("Charles B. Abbage");
    suspeitos.push("Donald Duck Knuth");
    suspeitos.push("Ada L. Ovelace");
    suspeitos.push("Alan T. Uring");
    suspeitos.push("Ivar J. Acobson");
    suspeitos.push("Ras Mus Ler Dorf");

    return suspeitos
}

function getLocais(){
    const locais = []

    locais.push("Redmond");
    locais.push("Palo Alto");
    locais.push("San Francisco");
    locais.push("Tokio");
    locais.push("Restaurante no Fim do Universo");
    locais.push("São Paulo");
    locais.push("Cupertino");
    locais.push("Helsinki");
    locais.push("Maida Vale");
    locais.push("Toronto");

    return locais
}

function getArmas(){
    const armas = []

    armas.push("Peixeira");
    armas.push("DynaTAC 8000X");
    armas.push("Trezoitão");
    armas.push("Trebuchet");
    armas.push("Maça");
    armas.push("Gládio");
    
    return armas
}

module.exports = {getSuspeitos, getLocais, getArmas};



